### DISCUSSION TITLE
How to uninstall Snip ?

### DISCUSSION THREAD

**Discussion by Zigomar7** [dlrudie/Snip#509] [Q&A] [answered]
Hi
I read that Snip would work with VLC in an old article (2016) so I went to DL (8.0.0.0), but that version don't show VLC as an alternative, can I use an older version (which) :
+ Looking at version history, I read that Snip v6.0.5 (2017) "fixed some VLC code", so it must have had VLC working, no other remarks in later versions, which don't mean explicitly that it is the last one where it is available. And would that last code, once located, still work correctly with today's VLC (2022) ?
+ If VLC is no longer supported, how can I uninstall Snip (It is not showing in Apps on my PC).
+ Is there a software alternative to Snip that will support VLC (I'm not using nor planning on using Spotify or iTunes).

Thanks
Zig

**Comment by dlrudie** [comment, marked as answer]
Just delete the directory.
